### PR TITLE
fix(FEV-933): fix "Hide button" location

### PR DESCRIPTION
**Pull request by semarche-kaltura**
Current PR fixes bug with "hide" button that currently has absolute position and overlapped top bar when PIP component had top-right or top-left position.
The changes discussed with @RoyBregman: "hide" button use relative position and located "inside" draggable container (see on attached screenshots)

For screenshots background of **playkit-interactive-area** changed to **red**

Screen Shot 2021-06-17 at 5 00 03 PM
Screen Shot 2021-06-17 at 4 58 43 PM

### CODE REVIEW

**Comment by yairans** [comment]
@semarche-kaltura please describe the solution in the PR description block

File: src/components/pip/pip.scss [hunk 1]
```diff
@@ -47,9 +49,17 @@ $hideContainerGap: 5px;
     }
   }
 
+  .videoContainer {
```
Comment (RoyBregman) [inline src/components/pip/pip.scss:52]: Please rename video to player as we will have also slides in the very near future

**Comment by RoyBregman** [comment]
Code looks good but behaviour is not so good:
When starting to drag it "jumps" and also when dragging and snapping to one of the bottom corners the animation got bad (In Master it is perfect)
Look:

https://user-images.githubusercontent.com/48884909/122725308-850e8b80-d27d-11eb-8c5d-e3838a80ab1c.mov


https://user-images.githubusercontent.com/48884909/122725348-8fc92080-d27d-11eb-99d2-8305960fb7af.mov

File: src/components/pip/pip.scss (hunk 1 above)
Comment (semarche-kaltura) [reply src/components/pip/pip.scss:52]: done

**Comment by semarche-kaltura** [comment]
> Code looks good but behaviour is not so good:
> When starting to drag it "jumps" and also when dragging and snapping to one of the bottom corners the animation got bad (In Master it is perfect)
> Look:

fixed

File: src/components/pip-minimized/pip-minimized.scss [hunk 2]
```diff
@@ -28,10 +31,9 @@
 
     position: absolute;
     width: 66px;
-    height: 32px;
+    height: $show-container-height;
     right: 0px;
-    top: 0px;
-    transform: translateY(calc(-100% - 5px));
+    top: calc((#{$show-container-height} + #{$show-container-gap}) * -1);
```
Comment (RoyBregman) [inline src/components/pip-minimized/pip-minimized.scss:36]:
```suggestion
    top: calc(-(#{$show-container-height} + #{$show-container-gap}));
```

File: src/components/pip/pip.scss [hunk 3]
```diff
@@ -7,26 +7,32 @@ $hideContainerGap: 5px;
...
+  }
+  &.dragging .hideContainer {
+    visibility: hidden;
   }
 
   .hideContainer {
+    position: relative;
+    width: 66px;
+    height: $hide-container-height;
+    margin-bottom: $hideContainerGap;
```
Comment (RoyBregman) [inline src/components/pip/pip.scss:27]: Please change casing of all vars in all scss files to "$xxxx-yyy-zzzzz".

**Review by RoyBregman — COMMENTED** [review]
image
Show container in pip-minimized is buggy now

File: src/components/pip/pip.scss [hunk 4]
```diff
@@ -47,9 +53,17 @@ $hideContainerGap: 5px;
...
+    position: relative;
+    border-radius: 8px;
+    overflow: hidden;
+    box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.2), 0 0 4px 0 rgba(0, 0, 0, 0.3);
+  }
+
   .innerButtons {
     flex-direction: row-reverse;
     border-radius: 8px;
+    z-index: 1;
```
Comment (RoyBregman) [inline src/components/pip/pip.scss:66]: Don't use z-index, instead - replace the order of the container and the inner buttons
image

File: src/components/pip-minimized/pip-minimized.scss (hunk 2 above)
Comment (semarche-kaltura) [reply src/components/pip-minimized/pip-minimized.scss:36]: no, it's invalid

File: src/components/pip/pip.scss (hunk 3 above)
Comment (semarche-kaltura) [reply src/components/pip/pip.scss:27]: done

File: src/components/pip/pip.scss (hunk 4 above)
Comment (semarche-kaltura) [reply src/components/pip/pip.scss]: done

**Review by RoyBregman — APPROVED** [review]

**Pull request merged**